internal/backend/svc/banner.txt

```diff
@@ -1,13 +1,13 @@
 
-        _..--~~~~--.._             autokitteh    {{ .Version }}
-     _--              --_          PID:          {{ .PID }}
-    '                    `         gRPC/HTTP:    {{ .Addr }}
-   '                      `        Web{{ .WebVer }}  {{ .WebAddr }}
-  '                        `       {{ .Mode }}
- |                    |\   |       
- |                    | \  |       {{ .Temporal }}
- |          .,    _,-~   \ |       {{ .TemporalUI }}
-  \         \ `.-'        \/ 
+        _..--~~~~--.._             autokitteh     {{ .Version }}
+     _--              --_          PID:           {{ .PID }}
+    '                    `         gRPC/HTTP API: {{ .Addr }}
```

Comment:
Web version: commented before but not fixed yet: you're making assumptions about the web ver string that are neither obvious nor stable.